platform/qemu/sifive_u/config.mk

```diff
@@ -13,6 +13,9 @@ platform-cflags-y =-mabi=lp64 -march=rv64imafdc -mcmodel=medany
 platform-asflags-y =-mabi=lp64 -march=rv64imafdc -mcmodel=medany
 platform-ldflags-y =
 
+platform-runcmd = qemu-system-riscv64 -M sifive_u -m 256M -display none -serial stdio \
+        -kernel build/platform/qemu/sifive_u/firmware/fw_payload.elf
```

Comment:
Build directory can be changed using "O=" command-line parameter to top-level makefile.

Better use $(build_dir)/platform/qemu/sifive_u/firmware/fw_payload.elf instead of build/platform/qemu/sifive_u/firmware/fw_payload.elf